website/cue/reference/components/sinks/base/appsignal.cue

```diff
@@ -198,11 +198,16 @@ base: components: sinks: appsignal: configuration: {
 				}
 			}
 			concurrency: {
-				description: "Configuration for outbound request concurrency."
-				required:    false
+				description: """
+					Configuration for outbound request concurrency.
+
+					This can be set either to one of the below enum values or to a uint, which denotes
```

Comment:
```suggestion
					This can be set either to one of the below enum values or to a uint, which denotes
```

Just a general comment/ question, would it be preferred here to change `uint` to `unsigned integer` just to avoid confusion on the acronym? Will approve the PR but leave that to your discretion.